go.mod

```diff
@@ -13,11 +13,11 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/onosproject/config-models/models/testdevice-1.0.x v0.5.29
 	github.com/onosproject/helmit v0.6.19
-	github.com/onosproject/onos-api/go v0.10.5
-	github.com/onosproject/onos-lib-go v0.9.5
+	github.com/onosproject/onos-api/go v0.10.22
+	github.com/onosproject/onos-lib-go v0.10.4
 	github.com/onosproject/onos-ric-sdk-go v0.8.9
 	github.com/onosproject/onos-test v0.6.6
-	github.com/onosproject/onos-topo v0.10.3
+	github.com/onosproject/onos-topo v0.10.6
```

Comment:
we should remove this dep. I can do it with removing unit tests associated with the client. I think that kind of unti testing is done in onos-topo itself and doesn't add anything extra in this repo.